### DISCUSSION TITLE
Shuffle Slack app

### DISCUSSION THREAD

**Discussion by mirceacroitoru** [Shuffle/Shuffle#1078] [Q&A]
Hi all
I configured Shuffle to send messages to Slack. This works.
However i am struggling to make the message more friendly. 
By default whatever is wrriten in Message box it's displayed continuosly.
What is the character for creating a new line? To have messages split into 2 lines.

**Comment by mirceacroitoru** [comment]
The message box shows like this

Test Firewall $repeat.all_fields.data.devname

Username: $repeat.all_fields.data.dstuser
Remote IP: $repeat.all_fields.data.remip    

Time: $repeat.all_fields.predecoder.timestamp

Rule ID:$repeat.rule_id

If i expand it shows below expected output however in Slack the message is continous not having new lines


This is how it looks in Slack.
The bold text like Agent, Location, Rule ID starts from new line in a slack message coming from a different system.
However from Shuffle it shows continuouly not having new lines ( the last line of the screenshot )

![image](https://user-images.githubusercontent.com/104731554/229277265-80e7d8f1-2a14-4126-97d7-0bca23ff9cef.png)

**Comment by frikky** [comment]
Hey! Looks like you'll have to add newlines directly as such: 
```
line 1\nline 2\nline 3
```

In certain cases it also seems like you may need to add a double backslash, but I'm unsure. It's all about the Slack API: https://api.slack.com/reference/surfaces/formatting#line-breaks

Hope this helps :)

**Reply by mirceacroitoru** [reply]
awsome! thank you a lot
